### PR TITLE
Unused api endpoints deprecation

### PR DESCRIPTION
**Pull request by Jeday**
### Description

- added `sunsetBy` middleware to deprecate endpoints according to draft web-standards
- proposed default sunset date that is not a middle of the night
- marked endpoints for deprecation 

### Code review notes

- [Blog about headers](https://blog.stoplight.io/deprecating-api-endpoints)
- [Warning header spec](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Warning) 
- [Draft about deprecation](https://datatracker.ietf.org/doc/html/draft-ietf-httpapi-deprecation-header)

### Testing notes
For:
 - /api/eth-apr
 - /api/sma-steth-apr
 - /api/ldo-stats
 - /api/lido-stats
 - /api/lidostats
 
Response status: 299
Headers:
- Deprecation
- Sunset
- Link(for eth-api,sma-steth-apr)

### Checklist:

- [x] Checked the changes locally.
- [ ] Created / updated analytics events.
- [ ] Created / updated the technical documentation (README.md / [docs](https://docs.lido.fi/) / etc.).
- [ ] Affects / requires changes in other services (Matomo / Sentry / CloudFlare / etc.).

### CODE REVIEW

**Review by Tarens2 — DISMISSED** [review]

File: consts/api.ts [hunk 1]
```diff
@@ -3,6 +3,15 @@ export const ETHPLORER_TOKEN_ENDPOINT =
 
 export const HEALTHY_RPC_SERVICES_ARE_OVER = 'Healthy RPC services are over!';
 
+// 5rd August, Monday, middle of the working day
+export const API_DEFAULT_SUNSET_TIMESTAMP = new Date(
+  '2024-08-05T09:00:00',
+).getTime();
+
+export const MAINNET_REPLACEMENT_LINKS: Partial<Record<API_ROUTES, string>> = {
+  'api/eth-apr': 'https://eth-api.lido.fi/v1/protocol/eth/apr/last',
```
Comment (DiRaiks) [inline consts/api.ts:12]: lets use origin from ENV

Comment (Jeday) [reply consts/api.ts:12]: This pr is not merged yet.

**Review by solidovic — DISMISSED** [review]
LGTM. But it looks we should to fix tests.

**Review by DiRaiks — DISMISSED** [review]

File: consts/api.ts (hunk 1 above)
Comment (Jeday) [reply consts/api.ts]: fixed

**Review by DiRaiks — APPROVED** [review]

**Pull request merged**